docker/el7/Dockerfile

```diff
@@ -15,8 +15,17 @@ RUN yum -y install /rpms/onearth-*.el7.*.rpm && yum clean all
 WORKDIR /pip/
 COPY pip-requirements.txt .
 RUN pip install --upgrade pip && pip install -r pip-requirements.txt
```

Comment:
Do we still need pip (python2 pip) for anything? I think we should just use pip3 everywhere.

Reply:
I didn't try removing it, but I would suspect that GDAL is still using it.  I thought we would remove the python2 after the python3 is validated by the testers.  I could try removing it if you want, but I figured it could be for a later ticket after we figure out the GDAL upgrade.

Reply:
I would try removing it and see if anything breaks. I don't think we should be using it anywhere.